src/client/pps.go

```diff
@@ -158,7 +158,7 @@ func NewUnionInput(input ...*pps.Input) *pps.Input {
 
 // NewCronInput returns an input which will trigger based on a timed schedule.
 // It uses cron syntax to specify the schedule. The input will be exposed to
-// jobs as `/pfs/<name>/time` which will contain a timestamp.
+// jobs as `/pfs/<name>/<timestamp>`.
```

Comment:
Would you mind adding a mention of the format we're using for this timestamp in here so people know how to parse it. (We probably should have had that in the old version of the code as well, but alas.)